syntax="proto3";

package full_stack_web;

message database_config {
	string host = 1;
	string user = 2;
	string passwd = 3;
	string db = 4;
	int32 port = 5;
	string socket = 6;
	bool skip_db_creation = 7;
}
